presto-client/src/main/java/io/prestosql/client/QueryError.java

```diff
@@ -31,6 +31,7 @@
     private final String errorType;
     private final ErrorLocation errorLocation;
     private final FailureInfo failureInfo;
+    private final String semanticErrorName;
```

Comment:
Let's add this directly after `errorName`, since it provides more specific information about `errorName`. Change the ordering everywhere in this class.